x-pack/plugins/code/public/reducers/__tests__/match_container_name.test.ts

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
for jest test, we don't need to place it in `__tests__` folder. Also, maybe we can move the `matchContainerName` into the `utils` folder and import it into the reducer?

Reply:
I guess to put test in `__tests__` would keep the file structure clean, and I'll move the function to  `utils`.